troubadix/standalone_plugins/changed_cves.py

```diff
@@ -0,0 +1,100 @@
+import re
+from argparse import ArgumentParser, Namespace
+from subprocess import CalledProcessError
+
+from pontos.terminal.terminal import ConsoleTerminal
+
+from troubadix.argparser import file_type
+from troubadix.helper.patterns import (
+    SpecialScriptTag,
+    get_special_script_tag_pattern,
+)
+from troubadix.standalone_plugins.common import get_merge_base
+
+from .changed_oid import git
```

Comment:
```suggestion
from troubadix.standalone_plugins.common import get_merge_base, git
```

:grey_question:

Reply:
Thanks, seems like I missed that after refactoring.